go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/doniacld/grpc-in-go
+
+go 1.18
```

Comment:
But you said 1.17 :fearful:  !

Reply:
I thought I was on Go 1.17 when developing.. 🧐